mistletoe/block_token.py

```diff
@@ -1020,6 +1020,111 @@ def read(lines):
         return [next(lines)]
 
 
+class HTMLAttributes(BlockToken):
```

Comment:
In the meantime, we switched to "strict/simple camel case convention" globally, so it needs to be like this now:

```suggestion
class HtmlAttributes(BlockToken):
```

The same applies to the new `HTMLAttributesRenderer`.